security/pkg/k8s/controller/casecret.go

```diff
@@ -49,7 +49,7 @@ func (csc *CaSecretController) LoadCASecretWithRetry(secretName, namespace strin
 	for {
 		caSecret, scrtErr = csc.client.Secrets(namespace).Get(context.TODO(), secretName, metav1.GetOptions{})
 		if scrtErr == nil {
-			return caSecret, scrtErr
+			return caSecret, nil
```

Comment:
This does not look like a related change for this PR. otherwise LGTM.

Reply:
This was because of the linting test failure in https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/32876/lint_istio/1393933255299829760 so I had to update to fix the test.